lib/super_good/solidus_taxjar.rb

```diff
@@ -45,6 +46,12 @@ def api
     }
     self.line_item_tax_label_maker = ->(taxjar_line_item, spree_line_item) { "Sales Tax" }
     self.logging_enabled = false
+
+    # The reporting feature is still in development. We recommend *not*
```

Comment:
We should add a ticket to the reporting project to remove this comment, even if that ticket is blocked for a long time. If that work isn't tracked, I'm afraid this comment will never get removed.

Reply:
Instead of ticketing the removal of this comment, I think we should ticket changing the default configuration value of this setting to `true`. And maybe mention removing this comment as part of that ticket.

Reply:
I created #105.